plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -64,7 +64,7 @@ const (
 	// The scheduler uses the annotation to find that the pod shouldn't preempt more pods
 	// when it gets to the head of scheduling queue again.
 	// See podEligibleToPreemptOthers() for more information.
-	NominatedNodeAnnotationKey = "NominatedNodeName"
+	NominatedNodeAnnotationKey = "kubernetes.io/nominated-node-name"
```

Comment:
prefer `kubernetes.io/NominatedNodeName`

Reply:
@k82cn I think we should use a name consistent with other Kubernetes DNS 1123 names. We don't allow capital letters in the names. So, I think `kubernetes.io/nominated-node-name` is a better name.